Sources/Document/Json/JSONObject.swift

```diff
@@ -16,145 +16,222 @@
 
 import Foundation
 
-protocol JSONSpec {
-    init()
-    static var keyMap: [AnyKeyPath: String] { get }
-}
-
-protocol JSONObjectable: AnyObject {
-    var dataHandler: ObjectDataHandler? { get set }
-}
-
 /**
  * `JSONObject` represents a JSON object, but unlike regular JSON, it has time
  * tickets created by a logical clock to resolve conflicts.
  */
-@dynamicMemberLookup
-class JSONObject<T: JSONSpec>: JSONObjectable {
-    var dataHandler: ObjectDataHandler?
-    private var jsonSpec: T
-    private let keyMap: [AnyKeyPath: String]
+class JSONObject {
+    var target: CRDTObject!
+    var context: ChangeContext!
 
-    init() {
-        self.jsonSpec = T()
-        self.keyMap = T.keyMap
-    }
+    init() {}
 
     init(target: CRDTObject, context: ChangeContext) {
-        self.dataHandler = ObjectDataHandler(target: target, context: context)
-        self.jsonSpec = T()
-        self.keyMap = T.keyMap
+        self.target = target
+        self.context = context
     }
 
-    subscript<V>(dynamicMember member: WritableKeyPath<T, V>) -> V {
-        get {
-            guard let key = self.keyMap[member] else {
-                assertionFailure("Must define a label for keyPath\(String(describing: member))")
-                return self.jsonSpec[keyPath: member]
+    func set(_ values: [String: Any]) {
+        values.forEach { (key: String, value: Any) in
+            if let value = value as? [String: Any] {
+                set(key: key, value: JSONObject())
+                let jsonObject = get(key: key) as? JSONObject
+                jsonObject?.set(value)
+//            } else if let value = value as? [Any] {
+                // It will be implemented soon.
+            } else {
+                set(key: key, value: value)
             }
+        }
+    }
 
-            Logger.trivial("obj[\(key)]")
+    func set(key: String, values: [String: Any]) {
```

Comment:
The key could be "key1.key2" (include dot). 
As I mentioned at get(key:), set(key:) also extends to perform nested key values.

Or prevent key strings containing dot characters are needed.

Reply:
I modified it. If a dot is used for the key, it will print an error log.